### PR TITLE
Fixing checksum for some checks

### PR DESCRIPTION
**Pull request by oliver-huetkoeper**
Checksum calculation is now triggered by the check class. Therefore no fallback checksum (based on line numbers) is calculated. This is important for the ATC baseline feature.

### CODE REVIEW

File: src/checks/zcl_aoc_check_02.clas.abap [hunk 1]
```diff
@@ -99,6 +100,7 @@ CLASS ZCL_AOC_CHECK_02 IMPLEMENTATION.
 
     has_attributes = abap_true.
     attributes_ok  = abap_true.
+    uses_checksum  = abap_true.
 
     enable_rfc( ).
```
Comment (larshp) [inline src/checks/zcl_aoc_check_02.clas.abap]: I'm not sure if the `uses_checksum` feature works on lower releases, abapOpenChecks is targeted

suggest using the same approach as in method `enable_rf( )`

**Comment by larshp** [comment]
related to #647

**Comment by larshp** [comment]
@oliver-huetkoeper

**Comment by oliver-huetkoeper** [comment]
Yes you are right. I think, this only works for central check systems (>=7.51). 
I didn’t have the time to adjust the coding yet. The suggested approach sounds fine.

**Comment by oliver-huetkoeper** [comment]
I have adjusted the coding as suggested.

**Pull request merged**